Exception/FatalThrowableError.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Paysera\Bundle\RestBundle\Exception;
+
+use Throwable;
+use ParseError;
+use ErrorException;
+
+class FatalThrowableError extends FatalErrorException
```

Comment:
@PayseraGithub Please have a look to this line https://github.com/paysera/lib-rest-bundle/pull/30/files#diff-56a47e941f07f2a1b84afbab53747c6f0adadf3aa41b3a5c011ae5369fd5d9cfL255 and `$exception = $event->getException();` - this code block removed. This code block removed because it was marked as depricated `since Symfony 4.4, use getThrowable instead` here https://github.com/symfony/http-kernel/blob/4.4/Event/GetResponseForExceptionEvent.php#L57 and this one code line `return $this->exception ?? $this->exception = $this->throwable instanceof \Exception ? $this->throwable : new FatalThrowableError($this->throwable);` of method `public function getException() {}` here https://github.com/symfony/http-kernel/blob/4.4/Event/GetResponseForExceptionEvent.php#L59 So `FatalThrowableError` class used to keep SF 4.4 backward compability like in Symfony 4.4, it's a Symfony 4.4 way to process exceptions inside of `public function onKernelException {}` listener.